README.md

```diff
@@ -32,19 +32,17 @@ Think of it like apt/yum/homebrew for Kubernetes.
 
 ## Install
 
-Binary downloads of the Helm client can be found at the following links:
-
-- [OSX](https://kubernetes-helm.storage.googleapis.com/helm-v2.8.2-darwin-amd64.tar.gz)
-- [Linux](https://kubernetes-helm.storage.googleapis.com/helm-v2.8.2-linux-amd64.tar.gz)
-- [Linux 32-bit](https://kubernetes-helm.storage.googleapis.com/helm-v2.8.2-linux-386.tar.gz)
-- [Windows](https://kubernetes-helm.storage.googleapis.com/helm-v2.8.2-windows-amd64.tar.gz)
+Binary downloads of the Helm client can be found on the [Releases Page](https://github.com/kubernetes/helm/releases).
 
 Unpack the `helm` binary and add it to your PATH and you are good to go!
 
 If you want to use a package manager:
 
-- macOS/[homebrew](https://brew.sh/) users can use `brew install kubernetes-helm`.
-- Windows/[chocolatey](https://chocolatey.org/) users can use `choco install kubernetes-helm`.
+- [homebrew](https://brew.sh/) users can use `brew install kubernetes-helm`.
+- [chocolatey](https://chocolatey.org/) users can use `choco install kubernetes-helm`.
+- [GoFish](https://gofi.sh/) users can use `gofish install helm`.
+
+[homebrew](https://brew.sh/) users can use `brew install kubernetes-helm`, and [GoFish](https://gofi.sh/) users can use `gofish install helm`.
```

Comment:
it looks like this is a duplicate instruction? @bacongobbler

Reply:
oops good catch